src/components/Documents/DocumentTableRow.jsx

```diff
@@ -0,0 +1,68 @@
+// React Imports
+import React, { useContext } from 'react';
+// React Router Imports
+import { Link } from 'react-router-dom';
+// Material UI Imports
+import { useTheme } from '@mui/material/styles';
+import IconButton from '@mui/material/IconButton';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+// Context Imports
+import { DocumentListContext } from '../../contexts';
+import { StyledTableCell, StyledTableRow } from '../Table/TableStyles';
+
+/**
+ * DocumentTableRow Component - Component that generates the individual table
+ *
+ * @memberof Documents
+ * @name DocumentTableRow
+ * @param document
+ * @returns {React.ReactElement}
+ */
+
+const DocumentTableRow = ({ document }) => {
+  const theme = useTheme();
+  const { removeDocument } = useContext(DocumentListContext);
+
+  const { name, type, description, fileUrl, uploadDate, endDate } = document;
+
+  // Event handler for deleting client from client list
+  const handleDeleteDocument = async () => {
+    if (
+      !window.confirm(
+        `You're about to delete ${document.name} from the pod, do you wish to continue?`
+      )
+    ) {
+      return;
+    }
+    await removeDocument(document.name);
+  };
+
+  return (
+    <StyledTableRow>
+      <StyledTableCell align="center">{name}</StyledTableCell>
+      <StyledTableCell align="center">{type}</StyledTableCell>
```

Comment:
For `type` think we could revert this back to the original string format with spacing. Would really just affect `types` with more than one word. Should be fine for the most part.

Reply:
updated